src/main/resources/application-baas.yml

```diff
@@ -16,6 +16,8 @@ bb-fuel:
     transactions: https://app.${environment.name}.${environment.domain}/transactions/integration-api
     user: https://app.${environment.name}.${environment.domain}/user-manager/integration-api
     accountStatement: https://app.${environment.name}.${environment.domain}/accountstatementsintegration/service-api
+    pockets: http://app.${environment.name}.${environment.domain}/api/pocket-tailor/client-api
+    pocketsArrangements: http://app.${environment.name}.${environment.domain}/pocketsprovidermock/service-api
```

Comment:
we need to check the baas deployment and add the appropriate mock service here and add it in baas deployment if doesn't exist there.

Reply:
I will add a TODO